org.eclipse.jdt.ui/ui/org/eclipse/jdt/ui/text/JavaSourceViewerConfiguration.java

```diff
@@ -457,6 +465,9 @@ public IContentAssistant getContentAssistant(ISourceViewer sourceViewer) {
 			ContentAssistProcessor javadocProcessor= new JavadocCompletionProcessor(getEditor(), assistant);
 			assistant.setContentAssistProcessor(javadocProcessor, IJavaPartitions.JAVA_DOC);
 
+			ContentAssistProcessor markdownProcessor= new JavadocCompletionProcessor(getEditor(), assistant);
+			assistant.setContentAssistProcessor(markdownProcessor, IJavaPartitions.JAVA_MARKDOWN_COMMENT);
+
```

Comment:
There's some inconsistency:
* the processor is registered for markdown partition
* the ctor of JavadocCompletionProcessor still associates it to the javadoc partition

When I tweak JavadocCompletionProcessor to pass the markdown partition into the super ctor, then no proposals will be shown in markdown any more.

IIUC, the reason is that plugin.xml does not associate any proposal computer with the markdown partition type.

Suggestion: add a parameter `partition` to the ctor of JavadocCompletionProcessor, then selectively add markdown partition in plugin.xml. This should give us control to enable desired processors while excluding, e.g., HTMLTagCompletionProposalComputer.

Reply:
In order to boost my confidence in this area (and to get this code to a build), I will take this up also as part of my next iteration.